lib/membrane/rtp/session_bin.ex

```diff
@@ -375,9 +384,31 @@ defmodule Membrane.RTP.SessionBin do
       }
     }
 
+    # workaround: as TWCC is a transport-wide extension, there should exist only one TWCC child
+    # that handles packets from all incoming streams that have declared support for it
+    {maybe_twcc, rtp_extensions} = Keyword.pop(rtp_extensions, :twcc)
+
+    should_link_twcc? = maybe_twcc != nil
+
+    {new_children, state} =
+      if should_link_twcc? and not Map.has_key?(ctx.children, :twcc) do
+        {twcc_ssrc, state} = add_ssrc(nil, state)
+
+        {Map.put(new_children, :twcc, %{maybe_twcc | sender_ssrc: twcc_ssrc}), state}
+      else
+        {new_children, state}
+      end
+
+    maybe_link_twcc =
+      &(&1
+        |> via_in(Pad.ref(:input, ssrc))
+        |> to(:twcc)
+        |> via_out(Pad.ref(:output, ssrc)))
+
```

Comment:
Can we put this whole logic to a separate function and just use it in pipe chain? Should be more readable.